.github/CODEOWNERS

```diff
@@ -114,12 +126,18 @@
 # PRLabel: %Monitor
 /sdk/loganalytics/                                                   @pvaneck
 
+# AzureSDKOwners: @pvaneck
```

Comment:
No need to duplicate the AzureSDKOwners and ServiceLabel for `Monitor`.   The association for issues is with the label and not the path, so any path (or no path) for a single entry will work.